src/content/pages/quiz.en.html

```
@@ -0,0 +1,58 @@
+<html><head></head><body><div class="container mx-auto px-4">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)